### DISCUSSION TITLE
Support Aria2 with download.

### DISCUSSION THREAD

**Discussion by tabris233** [Homebrew/discussions#3297] [Getting started]
Many times, brew download file by cURL, but cURL is too slow. 

Hope to support aria2 for download. Aria2 can download file faster than cURL.

**Comment by SMillerDev** [comment]
Please make a pull request to implement this functionality including some benchmarks to show the difference and the Homebrew maintainers will be happy to review it.

**Comment by alebcay** [comment]
See also a previous discussion at https://github.com/Homebrew/brew/issues/13064 about performing downloads in parallel (as a general enhancement, not even considering a lot of implementation details). This previous discussion also links to other discussions about the topic in the past.
